pkgs/top-level/python-packages.nix

```diff
@@ -4991,12 +4991,14 @@ self: super: with self; {
 
   jaxlib-bin = callPackage ../development/python-modules/jaxlib/bin.nix {
     cudaSupport = pkgs.config.cudaSupport or false;
+    cudaPackages = pkgs.cudaPackages_11_7;
   };
 
   jaxlib-build = callPackage ../development/python-modules/jaxlib rec {
     inherit (pkgs.darwin) cctools;
     # Some platforms don't have `cudaSupport` defined, hence the need for 'or false'.
     cudaSupport = pkgs.config.cudaSupport or false;
+    cudaPackages = pkgs.cudaPackages_11_7;
```

Comment:
Meaning here^^^
"# $(date -I): xxxx fails (how?) with cudaPackages_11_8, drop asap"

Or anything to your taste